src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_validators.py

```diff
@@ -170,6 +239,84 @@ def transform_acl_list_output(result):
         new_result.append(new_entry)
     return new_result
 
+def transform_cors_list_output(result):
+    new_result = []
+    for service in sorted(result.keys()):
+        service_name = service
+        for i, rule in enumerate(result[service]):
+            new_entry = OrderedDict()
+            new_entry['Service'] = service_name
+            service_name = ''
+            new_entry['Rule'] = i + 1
+            new_entry['Methods'] = ', '.join((x for x in rule['allowedMethods']))
+            new_entry['Origins'] = ', '.join((x for x in rule['allowedOrigins']))
+            new_entry['Exposed Headers'] = ', '.join((x for x in rule['exposedHeaders']))
```

Comment:
>'Exposed Headers [](start = 22, length = 16)

Header names should match the property names and be camel cased instead of having spaces.